scm-webapp/src/main/java/sonia/scm/api/v2/resources/IndexDtoGenerator.java

```diff
@@ -104,6 +104,7 @@ public IndexDto generate() {
         builder.single(link("config", resourceLinks.config().self()));
       }
       builder.single(link("repositories", resourceLinks.repositoryCollection().self()));
+      builder.single(link("namespaces", resourceLinks.namespaceCollection().self()));
```

Comment:
Is index the right place for namespaces? Is the repository collection not better suited for this link?

Reply:
Never mind, it looks like we use always the index e.g. repositoryVerbs, repositoryTypes, namespaceStrategies and repositoryRoles